packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2775,29 +2777,56 @@ export const audiusBackend = ({
     }
   }
 
+  /**
+   * Returns whether the current user is subscribed to userId.
+   * @returns Boolean
+   */
   async function getUserSubscribed(userId: ID) {
     await waitForLibsInit()
     const account = audiusLibs.Account.getCurrentUser()
     if (!account) return
-    try {
-      const { data, signature } = await signData()
-      return await fetch(
-        `${identityServiceUrl}/notifications/subscription?userId=${userId}`,
-        {
-          headers: {
-            [AuthHeaders.Message]: data,
-            [AuthHeaders.Signature]: signature
+
+    const readSubscribersFromDiscoveryEnabled =
+      (await getFeatureEnabled(
+        FeatureFlags.READ_SUBSCRIBERS_FROM_DISCOVERY_ENABLED
+      )) ?? false
+
+    if (readSubscribersFromDiscoveryEnabled) {
+      // Read subscribers from discovery
+      try {
+        const subscribers = await audiusLibs.User.getUserSubscribers(
+          encodeHashId(userId)
+        )
+        const subscriberIds = subscribers.map((s: APIUser) =>
```

Comment:
personal nit, i like still doing `subscriber` for the map argument, what you think?

Reply:
sure, added in 37d3ffce99d3ea6ae2137e569042e69810a15b64